src/app/containers/IndexPageSection/index.jsx

```diff
@@ -233,6 +241,7 @@ IndexPageSection.propTypes = {
   bar: bool,
   group: shape(groupShape).isRequired,
   sectionNumber: number.isRequired,
+  renderWithoutStrapline: number.isRequired,
```

Comment:
This should be a `bool` instead of `number`

Reply:
We also need to add `renderWithoutStrapline: false, ` to the `IndexPageSection.defaultProps` section